com.github.kevinsmia1939.PySimpleCV.yaml

```
@@ -0,0 +1,98 @@
+app-id: com.github.kevinsmia1939.PySimpleCV
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+command: PySimpleCV
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11 #PySimpleGUI Wayland not support
```

```suggestion
  - --socket=x11 #PySimpleGUI Wayland not support
```

obviously you didn't try on a wayland session.